pkg/cluster/generateconfig.go

```diff
@@ -100,6 +100,11 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 		workerZones = []string{""}
 	}
 
+	SDNProvider := "OVNKubernetes" // or "OpenShiftSDN", this will be the default SDNProvider
```

Comment:
I think we can drop the comment: it should be clear that `"OVNKubernetes"` is default and it is possible to override it.

Reply:
Dropped.